src/main/java/me/moodcat/database/MockData.java

```diff
@@ -14,36 +14,37 @@
 import com.google.inject.persist.Transactional;
 
 /**
- * TODO: Add explanation.
+ * MockData inserts initial data in to a clean database
```

Comment:
![](https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcTY4kSqVDiPpnOzN8_Pgkt4V5NUyyhTlR1DTHujPYYKL-zm9AWhjw)

But seriously, maybe we want to emphasise this is MOCK data. should be obvious by the filename, but still.